site/content/docs/0.0/migration-from-boosted.md

```diff
@@ -256,6 +283,23 @@ All responsive classes, helpers, and utilities have been updated accordingly to
     1. **Mixing Bootstrap numbers**: Search for `([" \.][mp]|-?gap)[tblrsexy]?-[a-z]{0,3}-?n?\d`, when you match multiple on one line, consider using one of our [scaled spacing utilities]({{< docsref "/utilities/spacing#notation-for-scaled-values" >}}) (for example, `.mb-1.mb-md-2` → `.mb-scaled-shortest`).
     2. **Remaining standalone classes**: Follow the previous guide going through all the standalone classes.
 
+### Text
```

Comment:
I'd add that `.ll-*` have been removed because all font references already have a max-width by default.